src/quadrature/Ordinate_Space.cc

```diff
@@ -322,9 +322,7 @@ void Ordinate_Space::compute_moments_(Quadrature_Class const quadrature_class,
     number_of_moments_ = 0;
     for (unsigned n = 0; n < moments_.size(); ++n) {
       int const l = moments_[n].L();
-      // member function prune() always returns true?
-      // if (l <= Lmax || !prune()) {
-      if (l <= Lmax) {
+      if (l <= Lmax || !prune()) {
         if (l > Lmax) {
           Lmax = l;
```

Comment:
Can you provide a test case that covers this branch for this block of code?

```
        if (l > Lmax) {
          Lmax = l;
          moments_per_order_.resize(Lmax + 1, 0U);
        }
```

This seems like a corner case and I don't understand what conditions would lead us here.